### ISSUE TITLE
libtiled-java cannot read tileset file in a jar

### ISSUE THREAD

**Issue by ZacharyJia** [bjorn/tiled#1231]
When I try to pack my project into a jar, the libtiled-java will throw an exception "canno't read input file" from ImageIO.read() method. I have tried some different format of source attribute but in vain.
Did I put an incorrect source of tileset ? or It's a bug?
Is there any solution to this problem?

**Comment by bjorn** [comment]
It's not currently a supported feature of libtiled-java.

Unfortunately I'm not doing any Java programming at all since some years and there is also a lot of other things to do on Tiled, so I haven't been getting around to working on libtiled-java much or at all. Pull requests that fix or extend this library are very much welcomed.

**Comment by ZacharyJia** [comment]
I have just made some changes to make it work for me, but I don't know if it would work for other situations. 
What I do is add an attribute 'jar', to the image label of the tileset manually, and do different operations to deal with the different situations.

**Comment by bjorn** [comment]
Right, would be nice if there could be some more automatic approach or something based on a bit of API changes, but I'm unsure how to do this and don't really have time to investigate.
